telegram/src/main/kotlin/com/github/kotlintelegrambot/entities/keyboard/InlineKeyboardButton.kt

```diff
@@ -45,6 +46,15 @@ sealed class InlineKeyboardButton {
         @SerializedName("switch_inline_query_current_chat") val switchInlineQueryCurrentChat: String
     ) : InlineKeyboardButton()
 
+    /**
+     * Description of the game that will be launched when the user presses the button.
+     * NOTE: this type of button must always be the first button in the first row.
```

Comment:
Pay buttons have the same constraint and to ensure library users don't use them incorrectly, there is some validation code in `InlineKeyboardMarkup` to ensure that pay buttons are the first button in the first row. Should we do the same for this type of button?

Reply:
@vjgarciag96 Ow I missed that, it's been fixed.
Also I took the liberty of refactoring the tests for this constraint with a Mothers factory for inlineKeyboardButtons.